### DISCUSSION TITLE
Data reporting different from Google Search Console queries

### DISCUSSION THREAD

**Discussion by andrewholgate** [plausible/analytics#1519] [Anything Else]
### Past Issues Searched

- [X] I have searched open and closed issues to make sure that the bug has not yet been reported

### Issue is a Bug Report

- [ ] This is a bug report and not a feature request, nor asking for self-hosted support

### Describe the bug

When comparing the daily data from Google Search Console user clicks and the Sources in Plausible, I'm noticing a slightly difference.

I've set the timezone in Plausible to Pacific Time (PT) to match that of Google Search Console. I've also connected GSC to Plausible.

Plausible in reporting higher values for users with the source of Google. Sometimes it's only a few % higher but other days it's 15% higher. What's consistent is that Plausible reports more.

I don't think this is a bug in Plausible but I was wondering if there might be an explanation for how this might be possible?

### Expected behavior

I'd expect Google Search Console clicks to be identical to the Google Sources in Plausible.

### Environment

```markdown
- OS:
- Browser:
- Browser Version:
```

**Comment by metmarkosaric** [comment]
hi @andrewholgate! these numbers are coming from two completely different sources so they are rarely identical. the one in our dashboard is counted using our own script so we include every visitor that comes to your site with google as the referrer header. we also include those visitors coming [from utm and other parameters that you titled Google](https://plausible.io/docs/manual-link-tagging#merging-organic-traffic-and-paid-campaigns). i'm not sure exactly how Search Console does it but i do know that they do a lot of sampling so they rarely show 100% exact data as it happens like we do. it's same if you compare Google Analytics and Search Console: it's rare that Google Analytics number of visitors from Google matches with the Search Console numbers. so yeah basically this is expected behavior as the data comes from two different sources with different tracking methods.

**Comment by andrewholgate** [comment]
Makes a lot of sense, thanks for the insights!
